Sources/App/Features/Locations/Migrations/Location+Migration+V1.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import Fluent
+import Vapor
+
+class LocationMigrationV1: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        try await database.schema(Schema.location)
+             .id()
+             .field("name", .string, .required)
+             .field("url", .string, .required)
+             .field("lat", .double, .required)
+             .field("long", .double, .required)
```

Comment:
It's a nitpick but should we name this "lon"

Reply:
I never actually know which is correct when they're shorter 😅 I'll whack up a change